mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+             - !ruby/object:Api::Type::Integer
+               name: 'threshold'
+               description: |
+                The minimum ratio of passing_rows / total_rows required to pass this rule, with a range of [0.0, 1.0]. 0 indicates default value (i.e. 1.0).
+               default_value: 0
+             - !ruby/object:Api::Type::NestedObject
+               name: 'rangeExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value lies between a specified range.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'minValue'
+                   description: |
+                     The minimum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::String
+                   name: maxValue
+                   description: |
+                     The maximum column value allowed for a row to pass this validation. At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMinEnabled'
+                   default_value: false
+                   description: |
+                     Whether each value needs to be strictly greater than ('>') the minimum, or if equality is allowed. 
+                     Only relevant if a minValue has been defined. Default = false.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMaxEnabled'
+                   default_value: false
+                   description: |
+                     Whether each value needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                     Only relevant if a maxValue has been defined. Default = false.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'nonNullExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value is null.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'setExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value is contained by a specified set.
+               properties:
+                 - !ruby/object:Api::Type::Array
+                   name: 'values'
+                   description: |
+                     Expected values for the column value.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'regexExpectation'
+               description: |
+                 ColumnMap rule which evaluates whether each column value matches a specified regex.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'regex'
```

Comment:
we should make this `required: true` since its the only nested attribute

Reply:
Same comment as above; unlikely but potential for new fields with one-of semantics with the regex field.